subjects/devops/code-keeper/README.md

```diff
@@ -220,6 +220,7 @@ you can implement anything that you feel deserves to be a bonus, for example:
 - Add `Infracost` in your infrastructure pipeline to estimate the
   infrastructure cost.
 - Use `Terragrunt` to create multiple Environments.
+- Use your own `crud-master` source code and `cloud-design` infrastructure.
```

Comment:
using cloud-design infrastructure is mandatory , since we are not providing the solution

Reply:
Hi @zamazzal, I see, but at the moment we don't have any `terraform` / `cloud-design` solution to provide the student: is there a way we could provide a starter code that does not include any IaC? Should we provide the solution of `orchestrator` instead of `crud-master`?

Reply:
I have updated the text as we discussed in our meeting today